zipline-loader/src/nativeTest/kotlin/app/cash/zipline/loader/URLSessionZiplineHttpClientTest.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.loader
+
+import kotlin.test.Test
+import kotlinx.coroutines.runBlocking
+import platform.Foundation.NSURLSession
+
+class URLSessionZiplineHttpClientTest {
+  @Test
+  fun makeRequests(): Unit = runBlocking {
```

Comment:
Just using this harness to see what happens. I need MockWebServer in multiplatform! 

(FYI @yschimke!)